src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -30,7 +30,9 @@ public static int Main(string[] args)
         public static int MainInternal(string[] args, CommandOutputLogger log)
         {
 #if DEBUG
-            if (args.Contains(DebugOption))
+            var debugNuGetXPlat = Environment.GetEnvironmentVariable("DEBUG_NUGET_XPLAT");
+
+            if (args.Contains(DebugOption) || string.Equals(bool.TrueString, debugNuGetXPlat, StringComparison.OrdinalIgnoreCase))
```

Comment:
yay! 👍 

I should have commit this a long time ago when I was working on the xplat cli project.